### ISSUE TITLE
Negative value

### ISSUE THREAD

**Issue by NingkangYang** [Maghoumi/pytorch-softdtw-cuda#27]
When the input and labels have only one feature and the values for both of them are between 0 and 1, the loss will be negative values.

**Comment by Maghoumi** [comment]
Yes this is expected and was discussed previously in #9 and #10.

Leaving this issue open for anyone else in the future who might have the same question.
